scipy/stats/tests/generate_fisher_exact_results_from_r.R

```diff
@@ -0,0 +1,82 @@
+# This R script generates the Python file fisher_exact_results_from_r.py.
+# The Python file is used in test_fisher_exact.py to compare the results
+# computed by scipy.stats.fisher_exact to those computed by R.
+
+sink("fisher_exact_results_from_r.py")
+
+options(digits=15)
+
+# Note that when converted to a matrix, `byrow=TRUE` is used, so
+# the first two values of each data set become the first row of
+# the contingency  table.  E.g.
+# > matrix(c(100, 2, 1000, 5), nrow=2, ncol=2, byrow=TRUE)
+#      [,1] [,2]
+# [1,]  100    2
+# [2,] 1000    5
+# 
+table_data = list(
```

Comment:
Where do these values come from?

Reply:
(Getting back to this PR--you might have to jog your memory a bit.)

These are array that were tested in https://github.com/scipy/scipy/blob/d4789babc26abcf09130b411df10eb72f8401d54/scipy/stats/tests/test_stats.py#L407